default.nix

```diff
@@ -0,0 +1,18 @@
+{ rustPlatform
```

Comment:
Can you merge the contents of this file (and also the overlay file) into the flake itself? I dislike having three nix-files when there is so little I need them to do.